data/net/minecraft/world/level/levelgen/synth/SurfaceNoise.mapping

```diff
@@ -1,5 +1,11 @@
 CLASS net/minecraft/world/level/levelgen/synth/SurfaceNoise
+	COMMENT The noise sampler used during surface building.
+	COMMENT This value is used as the {@code noise} parameter of {@link net.minecraft.world.level.levelgen.surfacebuilder.SurfaceBuilder#apply}.
+	COMMENT
+	COMMENT Mojang samples this noise rather strangely, by passing in (blockX, blockZ, 0.0625, localX).
+	COMMENT Since the localX only ranges from [0, 16), it can lead to axis aligned artifacts across chunk borders visible in the surface material (MC-199343).
```

Comment:
Most of this class' Javadoc might actually be better off as part of the Javadoc for #getSurfaceNoiseValue. The reference to MC-199343 should also use an `@see` or `@link` with a link to `https://bugs.mojang.com/browse/MC-199343`.